modules/popup/css/lib/popup.scss

```diff
@@ -1,203 +1,120 @@
-.wdc-popup {
-  @include wdc-type();
-  @include wdc-type-body();
-  @include wdc-depth-3();
-  display: inline-block;
-  background-color: $_wdc-popup-bg-color;
-  padding: $wdc-spacing-s;
-  border-radius: 3px;
-  border: 1px solid $_wdc-popup-border-color;
-  position: relative;
-  width: max-content;
-  max-width: $_wdc-popup-max-width;
-
-  // Fallback for IE11
-  @media screen and (-ms-high-contrast: active),
-  screen and (-ms-high-contrast: none) {
-    width: auto;
-    min-width: $_wdc-popup-menu-min-width;
+@keyframes popupAnimationTopCenter {
+  from {
+    opacity: 0;
+    transform: translate(0, -50%);
   }
 
-  // Fallback for Edge and browsers that don't support max-content
-  @supports not (width: max-content) {
-    width: auto;
-    min-width: $_wdc-popup-menu-min-width;
+  to {
+    opacity: 1;
+    transform: translate(0);
   }
+}
 
-  &-right,
-  &-left,
-  &-top,
-  &-bottom {
-
-    &:before,
-    &:after {
-      content: ' ';
-      position: absolute;
-      border-style: solid;
-      border-color: transparent;
-    }
-
-    &:before {
-      border-width: $_wdc-popup-arrow-size + 1px;
-    }
-
-    &:after {
-      border-width: $_wdc-popup-arrow-size;
-    }
+@keyframes popupAnimationRightCenter {
+  from {
+    opacity: 0;
+    transform: translate(-50%, 0);
   }
 
-  &-right {
-    margin-left: $wdc-spacing-s;
-
-    &:before,
-    &:after {
-      top: $_wdc-popup-arrow-offset;
-      right: 100%;
-    }
-
-    &:before {
-      margin-top: -($_wdc-popup-arrow-size + 1px);
-      border-right-color: $_wdc-popup-border-color;
-    }
-
-    &:after {
-      margin-top: -$_wdc-popup-arrow-size;
-      border-right-color: $_wdc-popup-bg-color;
-    }
+  to {
+    opacity: 1;
+    transform: translate(0);
   }
+}
 
-  &-left {
-    margin-right: $wdc-spacing-s;
-
-    &:before,
-    &:after {
-      top: $_wdc-popup-arrow-offset;
-      left: 100%;
-    }
-
-    &:before {
-      margin-top: -($_wdc-popup-arrow-size + 1px);
-      border-left-color: $_wdc-popup-border-color;
-    }
+@keyframes popupAnimationLeftCenter {
+  from {
+    opacity: 0;
+    transform: translate(50%, 0);
+  }
 
-    &:after {
-      margin-top: -$_wdc-popup-arrow-size;
-      border-left-color: $_wdc-popup-bg-color;
-    }
+  to {
+    opacity: 1;
+    transform: translate(0);
   }
+}
 
-  &-top {
-    margin-bottom: $wdc-spacing-s;
+@keyframes popupAnimationBottomCenter {
+  from {
+    opacity: 0;
+    transform: translate(0, 50%);
+  }
 
-    &:before,
-    &:after {
-      top: 100%;
-      left: $_wdc-popup-arrow-offset;
-    }
+  to {
+    opacity: 1;
+    transform: translate(0);
+  }
+}
 
-    &:before {
-      margin-left: -($_wdc-popup-arrow-size + 1px);
-      border-top-color: $_wdc-popup-border-color;
-    }
+.wdc-popup {
+  position: relative;
+  background-color: $_wdc-popup-bg-color;
```

Comment:
Can you update this to depend on card like the react version does? It would be nice if we didn't have the card-like styling here duplicated. (i.e. `@include card-common()`)